app/views/index.xml

```diff
@@ -30,7 +30,7 @@
 					<ListItem itemId="sizefill" title="FILL vs SIZE"/>
 					<ListItem itemId="progress" platform="android" title="Circular Progress"/>
 					<ListItem itemId="wkwebview" platform="ios" title="WKWebView Example"/>
-					<ListItem itemId="uibutton" platform="ios" title="UIButton Example"/>
+					<ListItem itemId="uibutton" platform="ios" title="Button Example"/>
```

Comment:
The identifier is still incorrect. Change the filename and identifier from `uibutton` to `button`.